libminifi/src/processors/ExtractText.cpp

```diff
@@ -21,6 +21,7 @@
 #include <string>
 #include <memory>
 #include <set>
+#include <regex>
```

Comment:
I don't think this is supported for all versions we allow. Should be easy to check on u14 and u16

Reply:
Regex support was added in gcc4.9, u16 uses 5.3, so that's more than fine, u14 is although uses 4.8, so yes, it's not gonna work. 
The question here is urgency: if we need it out pretty soon, I naturally do it. 
In case we don't, u14 is EOL at the end of this month, so I don't think we are going to support it for long, most probably not with 1.0. In this case it can wait. 
I'm happy with both options, what's your opinion?

Reply:
U14 and U16 are just examples. In the case of U14, even if it is EOL that is not the only system that uses GCC 4.8. In any case we should honor what is listed in our public Readme

Reply:
Added 4.8 compatible implementation as well - verified both with the unit tests.